app/views/content_items/speech.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= content_for :page_class, @content_item.format.dasherize %>
+<%= content_for :title, @content_item.title %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title',
+        context: t("content_item.format.#{@content_item.document_type}", count: 1),
+        title: @content_item.title,
+        average_title_length: 'long' %>
+  </div>
+</div>
+
+<%= render 'shared/withdrawal_notice', content_item: @content_item %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/metadata',
+        from: @content_item.from_with_speaker,
+        first_published: @content_item.published,
+        last_updated: @content_item.updated,
+        part_of: @content_item.part_of,
+        see_updates_link: true,
+        other: {
+          @content_item.delivery_type => @content_item.delivered_on_metadata,
+          "Location" => @content_item.location
+        }
+      %>
+  </div>
+</div>
+
+<%= render 'shared/history_notice', content_item: @content_item %>
+<%= render 'shared/description', description: @content_item.description %>
+<%= render 'shared/sidebar_with_body', content_item: @content_item %>
+
+<%= render 'govuk_component/document_footer',
+    from: @content_item.from,
+    published: @content_item.published,
+    updated: @content_item.updated,
+    history: @content_item.history,
+    part_of: @content_item.part_of
+%>
```

Comment:
We do seem to have several different styles of closing `%>` through the project. I'm not overly bothered, but it might be worth a tidy up at some point.